war/package.json

```diff
@@ -9,18 +9,38 @@
     "url": "https://github.com/tfennelly"
   },
   "private": true,
+  "scripts": {
+    "copy-fonts": "cp -rp src/main/fonts/* src/main/webapp/css",
```

Comment:
Question: is Jenkins built on an environment where this `cp` command would fail?

Reply:
Yea. It's tested on windows.

Theres a webpack copy plugin that works really well

Reply:
I have added the webpack copy plugin. Thanks!